EVENT.md

```diff
@@ -943,6 +943,16 @@ any one reporting origin.
 Strawman: 100 distinct destination sites per-{source site, reporting origin},
 applied to all unexpired sources regardless of type at source time.
 
+#### Limiting the number of unique destinations per source site
+
+To further reduce the possibility of a history reconstruction attack, the browser can also limit the number of `destination` eTLD+1s represented by source-sites.
+
+Strawman: 200 distinct destination sites per-{source site, 1 minute}
```

Comment:
Please update the "denial of service section" to add this possible denial of service attack.

Reply:
i have added a call out in that section